src/App.tsx

```diff
@@ -47,15 +48,25 @@ const App = () => {
     },
   });
 
+  const contextValue = {
+    isLight,
+    handleChangeToLightMode: () => {
+      setIsLight(true);
+    },
+    handleChangeToDarkMode: () => {
+      setIsLight(false);
+    },
+  };
```

Comment:
context 로 모드 state와 handler까지 같이 다루는거 너무 좋은 것 같습니다! 

여기서 모드에 따라 핸들러를 만들어주신 부분이 특이하다고 느꼈는데, 이렇게 각각의 핸들러를 만들어주신 이유가 궁금해요! 
아래와 같이 하나의 핸들러로도 구현할 수 있을 것 같은데 말이지요 !_!
```suggestion
  const contextValue = {
    isLight,
    handleChangeMode: (mode : 'dark' | 'light') => {
      setIsLight(mode==='light' ? true : false);
    },
  };
```
이렇게 작성하면 두개의 핸들러를 들고다니지 않으면서도 
사용부에서 
```js
handleChangeMode('dark');
```
요렇게 사용할테니 의미를 직관적으로 파악하는데에도 문제 없을 것 같은데 !1!

Reply:
오 너무 좋습니다~ 수정했습니다 :)